### DISCUSSION TITLE
Triggerbot time

### DISCUSSION THREAD

**Discussion by fanat1cc** [danielkrupinski/Osiris#4001] [Ideas]
I want to add a check for triggerbot and maybe aimbot that they won't take affect until after a person has been visible on the screen for a certain ms time.
But I have no idea how to do this, if anyone can link me a video or code that would work even if editied or help out thx.

**Comment by Xsintashi** [comment]
delay shot on triggerbot?

**Reply by fanat1cc** [reply]
Basically but I want it so I can have it always activate then with a delay once active after seeing a player

Aimbot would work better for this when holding corners the aimbot moves to them right away I want this to be delayed by x time
